acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/utils/Eth2RequestUtils.java

```diff
@@ -533,4 +426,37 @@ private static SafeFuture<Bytes> signingRootFromSyncCommitteeUtils(
     return SafeFuture.of(
         () -> createSigningRoot.apply(ALTAIR_SPEC.getSyncCommitteeUtilRequired(slot)));
   }
+
+  public static Eth2SigningRequestBody createBlobSidecarRequest(final boolean isBlinded) {
+    final Spec spec = TestSpecFactory.createMinimal(SpecMilestone.DENEB);
+    final tech.pegasys.teku.spec.datastructures.state.ForkInfo tekuForkInfo =
+        Eth2RequestUtils.forkInfo().asInternalForkInfo();
+    final Fork tekuFork = new Fork(tekuForkInfo.getFork());
+    final tech.pegasys.web3signer.core.service.http.handlers.signing.eth2.ForkInfo forkInfo =
+        new tech.pegasys.web3signer.core.service.http.handlers.signing.eth2.ForkInfo(
+            tekuFork, tekuForkInfo.getGenesisValidatorsRoot());
+    final Bytes signingRoot;
+    final tech.pegasys.web3signer.core.service.http.handlers.signing.eth2.BlobSidecar blobSidecar;
+    if (isBlinded) {
+      // generate random blinded blobsidecar
+      final BlindedBlobSidecar tekuBlindedBlobSidecar =
+          new DataStructureUtil(spec).randomBlindedBlobSidecar();
+      signingRoot =
+          new SigningRootUtil(spec)
+              .signingRootForBlindedBlobSidecar(tekuBlindedBlobSidecar, tekuForkInfo);
+      blobSidecar = fromInternalBlindedBlobSidecar(tekuBlindedBlobSidecar);
+    } else {
+      // generate random blobsidecar
+      final BlobSidecar tekuBlobSidecar = new DataStructureUtil(spec).randomBlobSidecar();
+      signingRoot =
+          new SigningRootUtil(spec).signingRootForBlobSidecar(tekuBlobSidecar, tekuForkInfo);
+      blobSidecar = fromInternalBlobSidecar(tekuBlobSidecar);
+    }
```

Comment:
Still don't understand why we are testing two cases. They are the same as far as Web3Signer is concerned. Are we gaining anything but having the other test case?

Reply:
You are right, I was still thinking with regards to Teku side of things ... I'll fix the test case to test only "BlobSidecar"